### PR TITLE
Add .gitattributes, update composer.json

### PR DESCRIPTION
**Pull request by KarelWintersky**
Added `gitattributes` (don't export some files to installed composer package). Updated composer.json (removed old, very old contributors that added self for P-R)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,5 +23,9 @@
     "psr-0": {
       "Curl": "src/"
     }
+  },
+  "support": {
+    "source": "https://github.com/php-curl-class/php-curl-class",
```
Comment (amouhzi) [inline composer.json:28]: This library is a fork of php-curl-class from November 3, 2013. They are no longer linked anymore.

https://github.com/php-mod/curl/commit/747df2b934fc29ab1f5b710cf6c43374bf25f6ac

**Comment by amouhzi** [comment]
This library is a fork of php-curl-class from November 3, 2013. They are no longer linked anymore.

https://github.com/php-mod/curl/commit/747df2b934fc29ab1f5b710cf6c43374bf25f6ac

**Comment by KarelWintersky** [comment]
@amouhzi , you rejects .gitattributes? Why?

**Comment by KarelWintersky** [comment]
so, I can't update PR.

**Pull request closed without merging**